stacks/api/bucket/bucket.module.ts

```diff
@@ -9,43 +9,53 @@ import {BucketDataService} from "./bucket-data.service";
 import {BucketController} from "./bucket.controller";
 import {BucketSchemaResolver, provideBucketSchemaResolver} from "./bucket.schema.resolver";
 import {CUSTOM_TYPES} from "./bucket.schema.types";
-import {BucketService} from "./bucket.service";
+import {BucketService} from "./services/bucket.service";
 import {BucketCache, provideBucketCache} from "./cache";
 import {DocumentScheduler} from "./scheduler";
+import {HookModule} from "@spica-server/bucket/hook";
+import {ServicesModule} from "@spica-server/bucket/services/bucket.service.module";
```

Comment:
We need to change this import clause to import the reference from `@spica-server/bucket/services`